packages/main/src/TimePickerBase.ts

```diff
@@ -230,10 +239,59 @@ class TimePickerBase extends UI5Element {
 		return this.tempValue;
 	}
 
+	get _isPhone() {
+		return isPhone();
+	}
+
 	onTimeSelectionChange(e: CustomEvent<TimeSelectionChangeEventDetail>) {
 		this.tempValue = e.detail.value; // every time the user changes the time selection -> update tempValue
 	}
 
+	/**
+	 * Opens the picker.
+	 * @async
+	 * @public
+	 * @method
+	 * @name sap.ui.webc.main.TimePickerBase#openPicker
+	 * @returns {Promise} Resolves when the picker is open
+	 */
+	async openPicker() {
+		this.tempValue = this.value && this.isValid(this.value) ? this.value : this.getFormat().format(new Date());
+		const responsivePopover = await this._getPopover();
+		responsivePopover.showAt(this);
+	}
+
+	/**
+	 * Closes the picker
+	 * @public
+	 * @method
+	 * @name sap.ui.webc.main.TimePickerBase#closePicker
+	 */
+	async closePicker() {
```

Comment:
It's good to add return type of every public function

Reply:
Done